### PR TITLE
Update the fuzzing

### LINKED ISSUE #24: Update fuzzing
### Description

In order to make repo able to detect bugs or vulnerabilities at early stage, update/add fuzzing rules.

### Acceptance Criteria

- Update fuzzing rules
- CI should pass with the additional fuzzing rules

### PR DESCRIPTION
**Pull request by hrmhatef**
### What was the problem?

This PR resolves #24 

### How was it solved?
 - Change the app name based on the `lisk`

### How was it tested?

 - All test passed.

### CODE REVIEW

File: fuzzing/CMakeLists.txt [hunk 1]
```diff
@@ -51,7 +51,7 @@ set(SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../src")
 set(ETH_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../ethereum-plugin-sdk")
 
 add_compile_definitions(
-    APPNAME="PluginBoilerplate"
+    APPNAME="Lisk"
```
Comment (nagdahimanshu) [inline fuzzing/CMakeLists.txt:54]: We'll also need to update description at [L10](https://github.com/LiskHQ/lisk-app-plugin/blob/24-update-the-fuzzing/fuzzing/CMakeLists.txt#L10) to `Contract parser of Lisk plugin app`

File: fuzzing/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Fuzzing allows us to test how a program behaves when provided with invalid, unexpected, or random data as input.
 
-In the case of `app-plugin-boilerplate` we want to test the code that is responsible for handling the contract data.
+In the case of `lisk` we want to test the code that is responsible for handling the contract data.
```
Comment (nagdahimanshu) [inline fuzzing/README.md]: What do you think of updating it to `lisk-app-plugin` everywhere?

Comment (hrmhatef) [reply fuzzing/README.md]: Done.

File: fuzzing/CMakeLists.txt (hunk 1 above)
Comment (hrmhatef) [reply fuzzing/CMakeLists.txt:54]: Done.

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by shuse2 — APPROVED** [review]

**Pull request merged**